README.md

```diff
@@ -74,7 +74,7 @@ Given that you can specify an exact version for `SwiftGen` in your `Podfile`, th
 You can then invoke SwiftGen in your Script Build Phase using:
 
 ```sh
-$PODS_ROOT/SwiftGen/bin/swiftgen …
+"${PODS_ROOT}/SwiftGen/bin/swiftgen" …
```

Comment:
Could you also update the example at line #60 to match this notation? Would make things more consistent.